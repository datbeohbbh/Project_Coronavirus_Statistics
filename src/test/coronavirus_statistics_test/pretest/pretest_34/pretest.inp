50 100
q 1 12 332
q 1 26 63
q 1 24 153
q 1 16 242
? 6
q 1 46 288
? 4 1
q 2 46 6
? 7
? 4 2
q 1 33 277
? 4 5
? 4 2
? 4 3
q 1 1 177
q 1 42 294
? 4 6
? 4 8
? 4 2
q 1 34 459
? 4 9
q 1 45 358
q 1 40 35
q 1 14 119
? 4 6
q 1 48 143
? 7
q 1 32 123
? 6
q 1 22 167
q 3 12 330
? 7
q 2 12 1
? 6
? 6
q 3 24 21
q 1 22 375
? 6
? 6
? 6
? 4 13
q 3 45 286
q 3 14 48
? 4 11
q 3 32 49
q 1 35 300
? 4 9
q 2 33 272
q 1 28 140
? 4 14
? 4 10
q 2 14 2
? 4 7
q 3 40 25
q 1 9 234
q 1 43 401
q 2 48 104
q 2 34 282
? 4 2
q 3 16 139
q 1 47 57
? 4 14
q 1 16 223
q 2 46 256
q 3 14 8
q 2 24 6
? 4 4
q 1 16 184
q 3 46 1
? 4 4
q 1 12 66
q 1 21 247
? 4 17
q 2 33 2
? 4 19
? 4 11
q 1 19 482
? 6
q 2 14 31
q 1 2 27
? 7
q 2 28 21
? 4 15
q 1 10 327
? 4 21
? 4 17
? 4 5
? 4 4
? 4 2
? 4 19
? 4 11
? 7
? 6
? 4 2
? 4 13
? 4 17
? 7
? 4 19
? 6
? 4 3