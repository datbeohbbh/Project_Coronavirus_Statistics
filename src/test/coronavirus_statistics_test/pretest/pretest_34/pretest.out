Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 4 1 :
The answer is the city with the index 26 
- Information of the city 26 -
-> Number of current patients : 63
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 2 :
The answer is the city with the index 24 
- Information of the city 24 -
-> Number of current patients : 153
-> Has recovered : 0
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 46 
- Information of the city 46 -
-> Number of current patients : 282
-> Has recovered : 6
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 24 
- Information of the city 24 -
-> Number of current patients : 153
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 242
-> Has recovered : 0
-> Was dead : 0

Query ? 4 6 :
The answer is the city with the index 46 
- Information of the city 46 -
-> Number of current patients : 282
-> Has recovered : 6
-> Was dead : 0

Query ? 4 8 :
The answer is the city with the index 12 
- Information of the city 12 -
-> Number of current patients : 332
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 24 
- Information of the city 24 -
-> Number of current patients : 153
-> Has recovered : 0
-> Was dead : 0

Query ? 4 9 :
The answer is the city with the index 34 
- Information of the city 34 -
-> Number of current patients : 459
-> Has recovered : 0
-> Was dead : 0

Query ? 4 6 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 242
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 6 :
The city with the highest recovery is city with the index 46
Quantity: 6 people

Query ? 7 :
The city with the highest death is city with the index 12
Quantity: 330 people

Query ? 6 :
The city with the highest recovery is city with the index 46
Quantity: 6 people

Query ? 6 :
The city with the highest recovery is city with the index 46
Quantity: 6 people

Query ? 6 :
The city with the highest recovery is city with the index 46
Quantity: 6 people

Query ? 6 :
The city with the highest recovery is city with the index 46
Quantity: 6 people

Query ? 6 :
The city with the highest recovery is city with the index 46
Quantity: 6 people

Query ? 4 13 :
The answer is the city with the index 45 
- Information of the city 45 -
-> Number of current patients : 358
-> Has recovered : 0
-> Was dead : 0

Query ? 4 11 :
The answer is the city with the index 33 
- Information of the city 33 -
-> Number of current patients : 277
-> Has recovered : 0
-> Was dead : 0

Query ? 4 9 :
The answer is the city with the index 1 
- Information of the city 1 -
-> Number of current patients : 177
-> Has recovered : 0
-> Was dead : 0

Query ? 4 14 :
The answer is the city with the index 42 
- Information of the city 42 -
-> Number of current patients : 294
-> Has recovered : 0
-> Was dead : 0

Query ? 4 10 :
The answer is the city with the index 48 
- Information of the city 48 -
-> Number of current patients : 143
-> Has recovered : 0
-> Was dead : 0

Query ? 4 7 :
The answer is the city with the index 32 
- Information of the city 32 -
-> Number of current patients : 74
-> Has recovered : 0
-> Was dead : 49

Query ? 4 2 :
The answer is the city with the index 33 
- Information of the city 33 -
-> Number of current patients : 5
-> Has recovered : 272
-> Was dead : 0

Query ? 4 14 :
The answer is the city with the index 1 
- Information of the city 1 -
-> Number of current patients : 177
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 46 
- Information of the city 46 -
-> Number of current patients : 26
-> Has recovered : 262
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 46 
- Information of the city 46 -
-> Number of current patients : 25
-> Has recovered : 262
-> Was dead : 1

Query ? 4 17 :
The answer is the city with the index 42 
- Information of the city 42 -
-> Number of current patients : 294
-> Has recovered : 0
-> Was dead : 0

Query ? 4 19 :
The answer is the city with the index 43 
- Information of the city 43 -
-> Number of current patients : 401
-> Has recovered : 0
-> Was dead : 0

Query ? 4 11 :
The answer is the city with the index 24 
- Information of the city 24 -
-> Number of current patients : 126
-> Has recovered : 6
-> Was dead : 21

Query ? 6 :
The city with the highest recovery is city with the index 34
Quantity: 282 people

Query ? 7 :
The city with the highest death is city with the index 12
Quantity: 330 people

Query ? 4 15 :
The answer is the city with the index 1 
- Information of the city 1 -
-> Number of current patients : 177
-> Has recovered : 0
-> Was dead : 0

Query ? 4 21 :
The answer is the city with the index 43 
- Information of the city 43 -
-> Number of current patients : 401
-> Has recovered : 0
-> Was dead : 0

Query ? 4 17 :
The answer is the city with the index 21 
- Information of the city 21 -
-> Number of current patients : 247
-> Has recovered : 0
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 14 
- Information of the city 14 -
-> Number of current patients : 30
-> Has recovered : 33
-> Was dead : 56

Query ? 4 4 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 27
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 40 
- Information of the city 40 -
-> Number of current patients : 10
-> Has recovered : 0
-> Was dead : 25

Query ? 4 19 :
The answer is the city with the index 35 
- Information of the city 35 -
-> Number of current patients : 300
-> Has recovered : 0
-> Was dead : 0

Query ? 4 11 :
The answer is the city with the index 32 
- Information of the city 32 -
-> Number of current patients : 74
-> Has recovered : 0
-> Was dead : 49

Query ? 7 :
The city with the highest death is city with the index 12
Quantity: 330 people

Query ? 6 :
The city with the highest recovery is city with the index 34
Quantity: 282 people

Query ? 4 2 :
The answer is the city with the index 40 
- Information of the city 40 -
-> Number of current patients : 10
-> Has recovered : 0
-> Was dead : 25

Query ? 4 13 :
The answer is the city with the index 24 
- Information of the city 24 -
-> Number of current patients : 126
-> Has recovered : 6
-> Was dead : 21

Query ? 4 17 :
The answer is the city with the index 21 
- Information of the city 21 -
-> Number of current patients : 247
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 12
Quantity: 330 people

Query ? 4 19 :
The answer is the city with the index 35 
- Information of the city 35 -
-> Number of current patients : 300
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 34
Quantity: 282 people

Query ? 4 3 :
The answer is the city with the index 46 
- Information of the city 46 -
-> Number of current patients : 25
-> Has recovered : 262
-> Was dead : 1

